docs/src/components/home/Sponsors.tsx

```diff
@@ -0,0 +1,180 @@
+import * as React from 'react';
+import Avatar from '@material-ui/core/Avatar';
+import Box, { BoxProps } from '@material-ui/core/Box';
+import Container from '@material-ui/core/Container';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Link from '@material-ui/core/Link';
+import IconButton from '@material-ui/core/IconButton';
+import AddRounded from '@material-ui/icons/AddRounded';
+
+const DIAMONDs = [
+  {
+    logo: 'https://avatars3.githubusercontent.com/u/1287123?s=256',
+    name: 'Octopus Deploy',
+    description: 'Repetable relayable deployments.',
+  },
+  {
+    logo: 'https://avatars3.githubusercontent.com/u/8424863?s=256',
+    name: 'Doit International',
+    description: 'Management platform for Google Clound and AWS.',
+  },
+];
+const GOLDs = [
+  {
+    logo: 'https://github.com/tidelift.png?size=96',
+    name: 'Tidelift',
+    description: 'Enterprise-ready open source software.',
+  },
+  {
+    logo: 'https://github.com/teambit.png?size=96',
+    name: 'Bit',
+    description: 'The fastest way to share code.',
+  },
+  {
+    logo: 'https://images.opencollective.com/callemall/a6946da/logo/96.png',
+    name: 'Text-em-all',
+    description: 'The easy way to message your group.',
+  },
+  {
+    logo: 'https://images.opencollective.com/canadacasino/5b19004/logo/96.png',
+    name: 'Canada Casino',
+    description: 'Safe and rewarding online casino experience',
+  },
+];
+
+function Label({
+  color,
+  children,
+  sx,
+  darker = false,
+}: {
+  color: 'primary' | 'warning';
+  children: React.ReactNode;
+  sx?: BoxProps['sx'];
+  darker?: boolean;
+}) {
+  return (
+    <Box
+      sx={{
+        borderBottomLeftRadius: 10,
+        borderBottomRightRadius: 10,
+        bgcolor: (theme) =>
+          theme.palette.mode === 'dark' ? `${color}.${darker ? '900' : '800'}` : `${color}.50`,
+        color: (theme) =>
+          theme.palette.mode === 'dark' ? `${color}.100` : `${color}.${darker ? '800' : '500'}`,
+        display: 'flex',
+        alignItems: 'center',
+        typography: 'caption',
+        fontWeight: 600,
+        p: 1,
+        ...sx,
+      }}
+    >
+      <Box
+        sx={{
+          display: 'inline-block',
+          mr: 1,
+          borderRadius: 1,
+          width: 8,
+          height: 8,
```

Comment:
I'm not sure if I'm commenting on the right place here, but the circles that signal the Diamond and Gold sponsors are too small. On my demo, I used 12px/12px and it ended up computing 14px/14px because of the border. We can make it bigger to 12 and see how it feels, maybe then changing to 14px.

Reply:
changed to 12px.